packages/qrcodescanner/QRScanner.js

```diff
@@ -0,0 +1,89 @@
+// Driver for the QR Code Scanner Unit
+// based from https://github.com/m5stack/M5Unit-QRCode/blob/main/src/M5UnitQRCodeI2C.cpp
+// Programmed by Ryoma Aoki
+
+const qrscanner_QRCODE_READY_REG = 0x0010;
+const qrscanner_QRCODE_LENGTH_REG = 0x0020;
+const qrscanner_QRCODE_TRIGGER_MODE_REG = 0x0030;
+const qrscanner_QRCODE_DATA_REG = 0x1000;
+const QRCODE_STATUS_READY = 1;
+const QRCODE_STATUS_DECODED = 2;
```

Comment:
[IMO]
定数に名前をつける場合は、一貫性を持たせた方が読み易いです。
```suggestion
const QRCODE_READY_REG = 0x0010;
const QRCODE_LENGTH_REG = 0x0020;
const QRCODE_TRIGGER_MODE_REG = 0x0030;
const QRCODE_DATA_REG = 0x1000;
const QRCODE_STATUS_READY = 1;
const QRCODE_STATUS_DECODED = 2;
```